bin/ssh_scan

```diff
@@ -127,6 +126,12 @@ scan") do |file|
   opts.on("-P", "--policy [FILE]",
           "Custom policy file (Default: Mozilla Modern)") do |policy|
     options["policy"] = policy
+    unless File.exist?(options["policy"])
+      puts opt_parser.help
+      puts "\nReason: policy file supplied is not a file"
+      exit 1
+    end
+    options["policy_file"] = SSHScan::Policy.from_file(options["policy"])
```

Comment:
This is a behavior change in which a user will always be required to specify a policy file.  Is this truly the behavior we're shooting for?